sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/util/NameAllocatorUtil.kt

```diff
@@ -0,0 +1,11 @@
+package com.squareup.sqldelight.util
+
+import com.squareup.javapoet.NameAllocator
+
+internal fun NameAllocator.getOrSet(objRef: Any, name: String): String {
```

Comment:
A better usage patter is to allocate your names up front. See square/wire for an example where this isn't a problem. We should adopt a similar pattern here to avoid this.